### DISCUSSION TITLE
Linux installation options

### DISCUSSION THREAD

**Discussion by dmcskim** [buttercup/buttercup-desktop#1294] [General]
I've noticed only AppImages are available for Buttercup installation. That makes me sad, I've enjoyed using your system for years. Will be deleting due to the lack of native installation options in Linux.

**Comment by perry-mitchell** [comment]
It's been like this for years already, so I'm not sure what's changed. The other options for electron based apps, like ours, have been poor. They haven't supported updating in-app, which was important for us, and so they were dropped. In the future I imagine we'll add them back, like deb/rpm and _maybe_ flatpak. But it's just me doing most of the maintenance so features get added as I get time. Appimage is fine for me so I haven't seen the need to add anything else just yet.

While I'm sorry to see you go, if this is the point you get hung up on then I won't try to persuade you otherwise. Good luck in your encrypting endeavours.

**Reply by dmcskim** [reply]
More than enough of a reason when there are dozens of free, secure alternatives available. Cheers.
